docs/examples/MarrmotM01.ipynb

```diff
@@ -2,196 +2,142 @@
  "cells": [
```

Comment:
<p>Notebook could be first introduction to Marrmot can we add link to https://github.com/wknoben/MARRMoT/</p><p> </p><p>As #150 is merged can you add link to https://ewatercycle.readthedocs.io/en/latest/system_setup.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>It is added.</p>